src/hooks/apiHooks.tsx

```diff
@@ -8,6 +8,34 @@ export interface LatestBeeReleaseHook {
   error: Error | null
 }
 
+export interface IsBeeDesktopHook {
+  isBeeDesktop: boolean
+  isLoading: boolean
+  error: Error | null
+}
+
+export const useIsBeeDesktop = (): IsBeeDesktopHook => {
+  const [isBeeDesktop, setIsBeeDesktop] = useState<boolean>(false)
+  const [isLoading, setLoading] = useState<boolean>(false)
+  const [error, setError] = useState<Error | null>(null)
+
+  useEffect(() => {
+    axios
+      .get(`${config.BEE_DESKTOP_URL}/info`)
+      .then(res => {
+        setIsBeeDesktop(true)
+      })
+      .catch((error: Error) => {
+        setError(error)
+      })
```

Comment:
Does it not cause any problems if we set this as an error instead of a soft boolean `false`?

Reply:
yeah I guess we can do that